ykrt/src/compile/jitc_yk/jit_ir.rs

```diff
@@ -1207,6 +1467,15 @@ impl Module {
         }
     }
 
+    /// Return the global declaration for the specified index.
+    ///
+    /// # Panics
+    ///
+    /// Panics if the index is out of bounds.
+    pub(crate) fn global_decl<'a>(&'a self, idx: GlobalDeclIdx) -> &'a GlobalDecl {
```

Comment:
This function also already exists in current master: https://github.com/ykjit/yk/pull/1031/commits/61daf62394110a0c87c5f77af66d8dcc4696f488#diff-d66ba8eec98a7d4ee4f268ff4459098b17150480892e69b748f2b50dfd5d2cb9R1311

Reply:
https://github.com/ykjit/yk/blob/master/ykrt/src/compile/jitc_yk/jit_ir.rs#L1067

Reply:
Fixed in 5b8c8f65f3972b294884b776ff56696c4dbf1570